.github/workflows/pr.yml

```diff
@@ -649,6 +649,145 @@ jobs:
           if [ "$numPass" -eq 0 ];then echo "*** 0 Passed! ***";exit 2;fi
           if [ "$numPass" -lt 26 ];then echo "*** Failed to pass at least 26 ! ***";exit 2;fi
           if [ "$numPass" -eq 0 ];then echo "*** Passed! ***";exit 0;fi
+  functional_docker_ubuntu_arm64:
```

Comment:
ensure it is added to both pr.yml and master.yml

Reply:
done